src/test/java/com/sfuentes/WorkerTest.java

```diff
@@ -0,0 +1,32 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class WorkerTest {
+
+  @Test
+  void calculateSalary_OneHour_TwoHundredPerHour() {
+    Worker test = new Worker(1, 1);
+    assertEquals(200, test.calculateSalary());
+  }
+
+  @Test
+  void calculateSalary_FortyHours_TwoHundredPerHour() {
+    Worker test = new Worker(1, 40);
+    assertEquals(8000, test.calculateSalary());
```

Comment:
here you didn't follow the same style in the Worjer.java file please add underscore to 8_000

Reply:
fixed.